wouso/core/magic/tests.py

```diff
@@ -301,36 +313,36 @@ def test_evade(self):
 
     def test_weakness(self):
         """
-         Test for weakness
+         Test for Weakness spell
         """
-
         initial_points = 10
 
-        player = self._get_player()
-        player2 = self._get_player(2)
-        chall_user = player.get_extension(ChallengeUser)
+        player_weakness = self._get_player(1).get_extension(ChallengeUser)
+        player_no_weakness = self._get_player(2).get_extension(ChallengeUser)
+        player_dummy = self._get_player(3).get_extension(ChallengeUser)
 
         scoring.setup_scoring()
         Coin.add('points')
-        scoring.score_simple(chall_user, 'points', initial_points)
-        self.assertEqual(player.points, initial_points)
+        scoring.score_simple(player_no_weakness, 'points', initial_points)
+        scoring.score_simple(player_weakness, 'points', initial_points)
+
+        # Apply weakness
+        weakness = Spell.objects.create(name='challenge-affect-scoring-won', available=True, price=10, percents=-66, type='p')
```

Comment:
@iulianR while penalties can be seen as negative rewards, having a spell that's been documented as negative implemented as a positive one with negative percents is not exactly right. We need to find a way to make it so it is -66, not + (-66).